src/subdomains/core/buy-crypto/process/services/buy-crypto.service.ts

```diff
@@ -77,7 +77,7 @@ export class BuyCryptoService {
     // transaction request
     entity = await this.setTxRequest(entity);
 
-    const multiAccountIban = await this.multiAccountIbanService.getAllMultiAccountIban();
+    const multiAccountIban = await this.specialExternalIbanService.getMultiAccountIban();
```

Comment:
`const multiAccountIbans = await this.specialExternalIbanService.getMultiAccountIbans();`